File: drama-service/src/main/java/example/drama/service/DramaServiceApplicationBoot.java
package example.drama.service;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@EnableDiscoveryClient
@SpringBootApplication
public class DramaServiceApplicationBoot {

    public static void main(String[] args) {
        new SpringApplicationBuilder(DramaServiceApplicationBoot.class).web(true).run(args);
    }

}

File: drama-service/src/main/java/example/drama/service/DramaServiceApplicationBoot.kt
package example.drama.service

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@SpringBootApplication
open class ApplicationBoot

fun main(args: Array<String>) {
    SpringApplicationBuilder(ApplicationBoot::class.java).web(true).run(*args)
}
